garak/resources/gcg/generate_gcg.py

```diff
@@ -36,7 +37,7 @@
 
 logger = getLogger(__name__)
 
-gpg_resource_data = garak._config.transient.basedir / "resources" / "gcg" / "data"
+gcg_resource_data = garak._config.transient.basedir / "resources" / "gcg" / "data"
```

Comment:
```suggestion
resource_gcg_data = garak._config.transient.basedir / "resources" / "gcg" / "data"
```

Another nitpick, just seems reasonable to match name to path.

Reply:
Again, going to defer this change to #479